src/style.css

```
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  max-width: 1200px;
+  min-width: 1024px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+  align-items: center;
+  line-height: 60px;
+}
+
+.box {
+  margin: 0 10px;
+  text-transform: uppercase;
+  align-items: center;
+  font-size: 12px;
+}
+
+.box:last-child {
+  margin-right: 0;
+}
+
+.box:hover {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  height: 4px;
+  width: 37px;
```

```suggestion
  width: 100%;
```
ми ж хочемо щоб це працювало для лінок будь-якого розміру?